test/integration/accessibility_spec.js

```diff
@@ -151,6 +151,22 @@ describe("accessibility", () => {
       await closePages(pages);
     });
 
+    it("must check the id in aria-controls", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.waitForSelector(".annotationLayer");
+          const stampId = "pdfjs_internal_id_20R";
+          await page.click(`#${stampId}`);
+
+          const controledId = await page.$eval(
+            "#pdfjs_internal_id_21R",
+            el => document.getElementById(el.getAttribute("aria-controls")).id
+          );
+          expect(controledId).withContext(`In ${browserName}`).toEqual(stampId);
```

Comment:
Spelling-nit:
```suggestion
          const controlledId = await page.$eval(
            "#pdfjs_internal_id_21R",
            el => document.getElementById(el.getAttribute("aria-controls")).id
          );
          expect(controlledId).withContext(`In ${browserName}`).toEqual(stampId);
```

Reply:
It was my first draft... and I thought that it doesn't make sense to have two 'l' here.